tests/test_cli.py

```diff
@@ -113,14 +114,17 @@ def test_command_list_node(workdir, mocker):
 @pytest.mark.parametrize('asset_name,params', [
     ('dataset', []),
     ('algo', []),
+    ('composite_algo', []),
     ('traintuple', ['--objective-key', 'foo', '--algo-key', 'foo', '--dataset-key', 'foo',
                     '--data-samples-path']),
     ('traintuple', ['--objective-key', 'foo', '--algo-key', 'foo', '--dataset-key', 'foo',
                     '--in-model-key', 'foo', '--data-samples-path']),
     ('traintuple', ['--objective-key', 'foo', '--algo-key', 'foo', '--dataset-key', 'foo',
                     '--in-model-key', 'foo', '--in-model-key', 'bar', '--data-samples-path']),
-    ('testtuple', ['--traintuple-key', 'foo', '--data-samples-path'])]
-)
+    ('testtuple', ['--traintuple-key', 'foo', '--data-samples-path']),
+    ('composite_traintuple', ['--objective-key', 'foo', '--algo-key', 'foo', '--dataset-key', 'foo',
+                              '--data-samples-path']),
```

Comment:
Could you add a test case with in_head/trunk_model_key? And a test checking that using only one of the two raises an error?